otwcode/otwarchive
# AO3-3207 Fix incorrect work blurbs and meta when series is updated

(Note: I've added `.vscode/` to `.gitignore`. Hope it's alright?) 

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-3207

## Purpose

Fix an issue when reordering series positions by drag and drop — this is caused by an incorrect datatype used in the AJAX request. Secondly, make the cache of works expire when a series is updated (especially when a new work is added to series). 

## Testing Instructions

See Jira issue and the past [comment by Lady Oscar](https://otwarchive.atlassian.net/browse/AO3-3207?focusedCommentId=335409).


## Credit
weeklies (she/her)

Huge thank you to **ticking_instant** and **Bilka** for answering my (beginner) questions on automated testing!
